src/Form/Mapper/FormMapper.php

```diff
@@ -20,7 +20,7 @@
  */
 interface FormMapper
 {
-    public function create(string $name, ?string $type = null, array $options = []): FormBuilderInterface;
+    public function create(string $name, ??string $type = null, array $options = []): FormBuilderInterface;
```

Comment:
:thinking:

Reply:
Bad regex replace :disappointed:
It should be fixed now.